.goreleaser.yml

```diff
@@ -10,7 +10,7 @@ before:
     - >-
       {{ if eq .Runtime.Goos "windows" }}echo{{ end }} make manpages GH_VERSION={{.Version}}
     - >-
-      {{ if ne .Runtime.Goos "linux" }}echo{{ end }} make completions
+      {{ if ne .Runtime.Goos "windows" }}echo{{ end }} make completions
```

Comment:
Could you explain this change?

Reply:
It was just for testing as i never worked with goreleaser before. I'll remove it.